plugins/regex_revalidate/regex_revalidate.c

```diff
@@ -362,7 +362,7 @@ config_handler(TSCont cont, TSEvent event ATS_UNUSED, void *edata ATS_UNUSED)
     iptr = __sync_val_compare_and_swap(&(pstate->invalidate_list), pstate->invalidate_list, i);
 
     if (iptr) {
-      free_cont = TSContCreate(free_handler, NULL);
+      free_cont = TSContCreate(free_handler, TSMutexCreate());
```

Comment:
I'm ok with this (assuming this solves the problem, it's not immediately clear as to why it needs the mutex). However, this approach of a schedule "delete" seems bad, we should change this (later) to use ref-counted data instead. That would eliminate the entire free_cont, and is a better / more reliable pattern.